socorro/unittest/external/postgresql/test_crontabber_state.py

```diff
@@ -42,7 +86,7 @@ def test_get(self):
                 '2013-02-09 00:16:00.893834',
                 '2012-12-24 22:27:07.316893',
                 6,
-                '{}',
+                '',
```

Comment:
In the old table, `depends_on` was a `text[]` and now it's just a `text`. So this adjusts the test accordingly.

There are no dependencies in our current crontabber apps, but I figured I'd keep the `depends_on` stuff for now since it was harder to rip all of it out.